src/components/graph/graph.ts

```diff
@@ -184,6 +206,25 @@ export default class GraphComponent {
     this._enableKeyboardPanning()
     this._enableNodeHighlightOnHover()
     this._disableDoubleClickZoom()
+
+    this._focusGraph()
+  }
+
+  private _focusGraph(): void {
```

Comment:
Feel free to refactor this if you think it can be improved. But I think this is the correct approach. We should not be relying on anything outside the scope of the graph otherwise it will lead to a poor design.